### PR TITLE
Add domain plugin

### PR DESCRIPTION
**Pull request by pseudokawaii**
Users will be able to list all the channels and groups they created or have administrative privileges on.

### CODE REVIEW

**Comment by Krishna-Singhal** [comment]
Nice, try combining both commands into 1 command.

**Comment by pseudokawaii** [comment]
@Krishna-Singhal how about now?

File: plugins/domain.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    },
+)
+async def creator(m: Message):
+
+    await m.edit("__This may take a while, please wait ...__")
+
+    if "-c" in m.input_str:
+        status = "creator"
+    elif "-a" in m.input_str:
+        status = "administrator"
```
Comment (Krishna-Singhal) [inline plugins/domain.py:22]: here also

File: plugins/domain.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            "-c": "List all gropus and channels created by you.",
+            "-a": "List all gropus and channels administered by you.",
+        },
+    },
+)
+async def creator(m: Message):
+
+    await m.edit("__This may take a while, please wait ...__")
+
+    if "-c" in m.input_str:
```
Comment (Krishna-Singhal) [inline plugins/domain.py:19]: use
```python
m.flags
```
 instead of 
```python
m.input_str
```

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

File: plugins/domain.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        status = "creator"
+    elif "-a" in m.input_str:
```
Comment (Krishna-Singhal) [inline plugins/domain.py:21]: here also

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

File: plugins/domain.py (hunk 3 above)
Comment (pseudokawaii) [reply plugins/domain.py]: @Krishna-Singhal done.

**Review by Krishna-Singhal — APPROVED** [review]

**Comment by rking32** [comment]
@pseudokawaii Thanks for this plugin :)

**Pull request merged**